CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **API Changes:**
 
 **Other:**
+- Replaced open func subscribe<SelectedState: Equatable, S: StoreSubscriber> with public func subscribe<SelectedState: Equatable, S: StoreSubscriber> and open func subscribe<S: StoreSubscriber>(_ subscriber: S) with public func subscribe<S: StoreSubscriber>(_ subscriber: S) since non-'@objc' instance method in extensions cannot be overridden, should use 'public' instead (#491) - @maksimgromov
```

Comment:
What do you think about the following to "hide" the details and focus on the intention?

```suggestion
- Replaced `open func` with `public func` in extensions to `Store` because they cannot be overridden anyway (#491) - @maksimgromov
```